lib/src/home.dart

```diff
@@ -90,3 +95,69 @@ class _HomeState extends State<Home> with SingleTickerProviderStateMixin {
     );
   }
 }
+
+class SimulationSearch extends SearchDelegate<SimulationCard> {
+  @override
+  List<Widget> buildActions(BuildContext context) {
+    return [
+      IconButton(
+        icon: Icon(Icons.clear),
+        onPressed: () {
+          query = '';
+        },
+      ),
+    ];
+  }
+
+  @override
+  Widget buildLeading(BuildContext context) {
+    return IconButton(
+      icon: Icon(Icons.arrow_back),
+      onPressed: () {
+        close(context, null);
+      },
+    );
+  }
+
+  @override
+  Widget buildResults(BuildContext context) {
+    return Container();
+  }
+
+  @override
+  Widget buildSuggestions(BuildContext context) {
+    final appState = Provider.of<Simulations>(context);
+    return (query != '')
+        ? (appState.searchSims(query).length != 0)
+            ? GridView.count(
+                crossAxisCount: (MediaQuery.of(context).size.width < 600)
+                    ? 2
+                    : (MediaQuery.of(context).size.width / 200).floor(),
+                children: appState.searchSims(query),
+              )
+            : Container(
+                child: Center(
+                  child: Text(
+                    "Sorry, couldn't find a simulation",
```

Comment:
I guess it would be better if this is not a centered widget. In the absence of the keyboard, it is centered; when the keyboard appears, it shifts upward.

Maybe we could fix a certain distance from the top?